components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -39,7 +39,9 @@
 #include "bat/ads/pref_names.h"
 #include "bat/ads/resources/grit/bat_ads_resources.h"
 #include "bat/ads/statement_info.h"
+#include "brave/browser/brave_adaptive_captcha/brave_adaptive_captcha_service_factory.h"
```

Comment:
this is a layering violation, you can't include brave/browser here

Reply:
Removed.